frontend/src/pages/landingPage/landingPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { groupBackground } from "../../assets/images/images";
+import { IconDownArrow } from "../../assets/images/images";
+import { Button } from "../../components/Buttons/Buttons";
+
+import "./landingPage.scss";
+
+function LandingPage() {
+  const LandingImage = () => {
+    return (
+      <div className="image-parent">
+        <div className="svg-group-layer">
+          <img className="group-image " src={groupBackground} />
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div className="text-center pt-5">
+      <div className="top-content">
+        <h1>
+          Together,
+          <br />
+          we can create greater civic change
+        </h1>
+        <p className="mb-5">
+          CivicTechJobs unites ambitious technology practitioners with volunteer
+          opportunities from a central hub of listings to build digital
+          products, programs, and services.{" "}
+        </p>
+        <Button addClass="action-btn">Join us</Button>
```

Comment:
This does not set the default size to lg, so it uses the default of sm. Check out the [button file ](https://github.com/hackforla/CivicTechJobs/blob/main/frontend/src/components/Buttons/Buttons.js)to see the props that it takes and [here ](https://github.com/hackforla/CivicTechJobs/blob/588db284358ab4be178fae87ec0c6235f0134474/frontend/src/components/App.js#L102)for sample usage.